tests/conftest.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+from sqlalchemy.engine import create_engine
+from sqlalchemy.orm.session import Session
+from geoalchemy2 import Geometry # NOQA
+
+from osmnames import settings
+from osmnames.init_database import init_database
+from osmnames.helpers.database import exec_sql
+
+
+@pytest.fixture(scope="module")
+def engine():
+    _recreate_database()
+
+    return create_engine("postgresql+psycopg2://{}:{}@{}/{}".format(
```

Comment:
This might be a good idea to show where the variables are being used.

```python
"postgis://{user}:{password}@{host}/{db_name}".format(
    user=settings.get("DB_USER"),
    password=settings.get("DB_PASSWORD"),
    host=settings.get("DB_HOST"),
    db_name=settings.get("DB_NAME"),
)
```